sdk/cdn/arm-cdn/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/arm-cdn",
   "author": "Microsoft Corporation",
   "description": "CdnManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "5.1.0",
+  "version": "6.0.0",
```

Comment:
Why are we updating the major version here? From what I can tell, we are changing some attributes from being mandatory to optional. This is not really a breaking change

Reply:
Agreed. @colawwj Please bump minor version here.